pkg/profiles/proto.go

```diff
@@ -77,3 +80,23 @@ func protoToServiceProfile(parser *proto.Parser, namespace, name, clusterDomain
 		},
 	}, nil
 }
+
+func isMethodRetryable(rpc *proto.RPC) bool {
+	for _, e := range rpc.Elements {
+		option, ok := e.(*proto.Option)
+		if !ok {
+			// Not an option
+			continue
+		}
+
+		if !strings.Contains(option.Name, "idempotency_level") {
```

Comment:
is there a reason this is a Contains rather than an equals comparison?  what does `option.Name` typically look like?

Reply:
Yes, that could have been more clear. The name can be in parentheses but also without those which is also why I have covered both cases in the unit tests.

Alternatively, we could trim away the parentheses to do an equals check instead of a contains check?

Reply:
Ah, I see.  Probably trimming the parentheses would be more correct.  Either way, I think this warrants a comment explaining.

Reply:
Great. I will add the comment and trim the parentheses. Thanks 👍🏼